build-logic/convention/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+plugins {
+    `kotlin-dsl`
+}
+
+group = "io.github.composegears.valkyrie"
+
+dependencies {
+    compileOnly(libs.kotlin.compose.compiler.plugin)
+    compileOnly(libs.kotlin.gradle.plugin)
+}
+
+gradlePlugin {
+    plugins {
+        register("valkyrie.kmp") {
+            id = "valkyrie.kmp"
+            implementationClass = "KmpPlugin"
+        }
+        register("valkyrie.compose") {
+            id = "valkyrie.compose"
+            implementationClass = "ComposePlugin"
+        }
+    }
```

Comment:
You can write them in `.gradle.kts` instead of `.kt`, no need to extend `Plugin` things.